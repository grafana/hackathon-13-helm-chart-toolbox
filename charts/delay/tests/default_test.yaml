# yamllint disable rule:document-start rule:line-length rule:trailing-spaces
suite: Delay - Default
templates:
  - tests/test-pod.yaml
tests:
  - it: creates a Pod to delay for two minutes
    set:
      delay: 120
    asserts:
      - isKind:
          of: Pod
      - matchSnapshot: {}
