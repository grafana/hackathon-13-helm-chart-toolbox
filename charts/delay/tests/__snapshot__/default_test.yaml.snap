creates a Pod to delay for two minutes:
  1: |
    apiVersion: v1
    kind: Pod
    metadata:
      annotations:
        helm.sh/hook: test
        helm.sh/hook-delete-policy: before-hook-creation
        helm.sh/hook-weight: "0"
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/version: 0.1.0
        helm.sh/chart: delay-0.1.0
      name: RELEASE-NAME-delay
      namespace: NAMESPACE
    spec:
      containers:
        - command:
            - sleep 120
          image: docker.io/library/busybox:latest
          name: delay
      nodeSelector:
        kubernetes.io/os: linux
      restartPolicy: Never
